IPython/core/prefilter.py

```diff
@@ -512,6 +512,8 @@ def check(self, line_info):
             callable(oinfo.obj)
             and (not self.exclude_regexp.match(line_info.the_rest))
             and self.function_name_regexp.match(line_info.ifun)
+            and line_info.raw_the_rest.startswith(" ")
+            or not line_info.raw_the_rest.strip()
```

Comment:
I think this introduces the bug reported in #14513 because this last `or` condition no longer requires the initial `callable(oinfo.obj)` from 512.